pom.xml

```diff
@@ -142,9 +142,9 @@
 
         <!-- Databases -->
         <mysql.version>8.0.23</mysql.version>
-        <mysql.driver.version>8.0.28</mysql.driver.version>
+        <mysql-jdbc.version>8.0.30</mysql-jdbc.version>
         <postgresql.version>13.2</postgresql.version>
-        <postgresql.driver.version>42.3.3</postgresql.driver.version>
+        <postgresql-jdbc.version>42.5.0</postgresql-jdbc.version>
         <mariadb.version>10.3.27</mariadb.version>
         <mariadb.driver.version>2.7.2</mariadb.driver.version>
         <mssql.version>2019-CU10-ubuntu-20.04</mssql.version>
```

Comment:
Just a note for a possible a follow-up, we might want to align also other DB drivers with Quarkus, not just mysql and postgres.

Reply:
Perhaps we can do it now too? Wdyt @Pepo48 ?

Are these versions expected by our internal pipeline?

Reply:
Thanks, I aligned the rest of the jdbc properties in a34d0f7. I didn't bump their versions though, since they don't have an direct affect in Quarkus module.

Reply:
@Pepo48 can you please make sure we have some tracker for this follow-up?

Reply:
@vmuzikar sure, I already informed QE.